Makefile

```
@@ -11,6 +11,9 @@ GOHOSTARCH ?= $(shell go env GOHOSTARCH)
 
 ENABLE_BTFGEN ?= false
 
+BPFTOOL ?= bpftool
+ARCH := $(shell uname -m | sed 's/x86_64/x86/' | sed 's/aarch64/arm64/' | sed 's/ppc64le/powerpc/' | sed 's/mips.*/mips/')
```

Should it be changeable by users like this?

```suggestion
ARCH ?= $(shell uname -m | sed 's/x86_64/x86/' | sed 's/aarch64/arm64/' | sed 's/ppc64le/powerpc/' | sed 's/mips.*/mips/')
```